### PR TITLE
update README to clarify the company_id requirement

### PR DESCRIPTION
**Pull request by tedyangx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,32 +92,44 @@ Stores automatically manage tokens for you - refreshing, revoking and storage
...
-options to work with [Multiple Procore Zones (MPZ)](https://developers.procore.com/documentation/tutorial-mpz).
+#### Requiring `company_id` in the `options`
 
-Example Usage:
+In order for your application to work with [Multiple Procore Zones (MPZ)](https://developers.procore.com/documentation/tutorial-mpz), each call your application makes to the Procore API **must** contain a request header that includes the `Procore-Company-Id` field. This request header field specifies the id of the company into which you are making the call.
+
+This can be achieved by setting the `company_id` in the request `options`. (And this gem will convert it into a `Procore-Company-Id` header automatically.)
+
+For example:
+```ruby
```
Comment (m-graf) [inline README.md:124]: I like these little details, thanks ted!

**Review by m-graf — APPROVED** [review]
This looks good!

File: README.md [hunk 2]
```diff
@@ -31,10 +30,56 @@ Add this line to your application's Gemfile:
...
 ```
 
-## 1.0.0 Release
+## Configuration
+
+The Procore Gem exposes a configuration with several options.
 
-v1.0.0 was released on January 5, 2021, and adds support the new Rest v1.0 API.
-See the CHANGELOG for upgrade instructions.
+For example, when use this gem in a Rails application, the configuration can be done like below:
```
Comment (martinr23) [inline README.md:37]:
```suggestion
For example, when using this gem in a Rails application, the configuration can be done like below:
```

**Review by martinr23 — APPROVED** [review]
🚀

**Pull request merged**